staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -522,6 +527,35 @@ func subsequentPatchLogic(p *patcher, ps patchSource, currentObject runtime.Obje
 	return p.toUnversioned(versionedObjToUpdate)
 }
 
+type applyPatcher struct {
+	*patcher
+}
+
+func (p *applyPatcher) firstPatchAttempt(currentObject runtime.Object, currentResourceVersion string) (runtime.Object, error) {
+	return p.structuredMerge()
+}
+
+func (p *applyPatcher) subsequentPatchAttempt(currentObject runtime.Object, currentResourceVersion string) (runtime.Object, error) {
```

Comment:
I thought these were deleted in Jordan's change--maybe we need to merge master back into our branch.

Reply:
Our feature branch is pretty far behind I think.